### PR TITLE
Use columnar result from clickhouse driver

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

**Comment by kszucs** [comment]
@cpcloud I guess some credentials are missing for bigquery. Could You give me a little help here? I've tried to disable circleci build on my account.

**Comment by cpcloud** [comment]
There's a PR open to fix this, let me merge that.

**Comment by cpcloud** [comment]
@kszucs Rebase on top of master and you should be good to go.

**Comment by kszucs** [comment]
I'm updating the clickhouse sqlalchemy recipe to depend on clickhouse driver >= 0.0.8, hopefully that'll resolve the dependency issue on python34 tests.

**Comment by kszucs** [comment]
@cpcloud ready for review

File: ibis/clickhouse/tests/test_client.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def test_result_as_dataframe(con, alltypes):
     result = con.execute(expr)
 
     assert isinstance(result, pd.DataFrame)
-    assert list(result.columns) == ex_names
+    assert sorted(result.columns) == sorted(ex_names)
```
Comment (cpcloud) [inline ibis/clickhouse/tests/test_client.py]: Is it possible that this fails without sorting?

Comment (kszucs) [reply ibis/clickhouse/tests/test_client.py]: It did previously, but I've fixed that later.

Comment (cpcloud) [reply ibis/clickhouse/tests/test_client.py]: Ok great!

**Comment by cpcloud** [comment]
@kszucs Can you rebase? I'll merge on green after that.

**Comment by kszucs** [comment]
@cpcloud rebase done

**Comment by cpcloud** [comment]
@kszucs Cool, I'll merge on a green build! Thanks!

**Comment by cpcloud** [comment]
Merging. This doesn't need a rebase because it's nothing touching anything except clickhouse related code.

**Comment by cpcloud** [comment]
Thanks @kszucs keep 'em coming!

**Pull request closed without merging**